src/app/components/LayerPicker/index.tsx

```diff
@@ -77,8 +77,8 @@ type LayerPickerContentProps = Omit<LayerPickerProps, 'open'>
 
 enum LayerPickerTabletStep {
   Network,
-  ParaTime,
```

Comment:
Doesn't ParaTime make more sense than Layer, and the LayerPicker should be renamed to ParaTime

Reply:
Why? We have consensus, and then we have the paratimes. Here, we can select both, not only the paratimes. The  term that covers both is layer, right?

Reply:
Ah you are right, I always forget Consensus :) Layer makes perfect sense.